ghost/sodo-search/scripts/build-combined.js

```diff
@@ -2,17 +2,6 @@ const rewire = require('rewire');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const defaults = rewire('react-scripts/scripts/build.js');
 let config = defaults.__get__('config');
-const fs = require('fs');
-/* eslint-disable no-console */
-const log = console.log;
-/* eslint-enable no-console */
-
-fs.copyFile('./public/main.css', './umd/main.css', (err) => {
```

Comment:
This file doesn't exist since https://github.com/TryGhost/Ghost/commit/21b621c4fa1578a57dcf94a2478607274a78e64d

I had to update this because it was messing up the ability to run `lerna run build` from monorepo root